tests/doxygen/main.md

```diff
@@ -0,0 +1,80 @@
+Introduction     {#mainpage}
+================
+
+[TOC]
+
+# Overview {#main-overview}
+
+Ja Rule is an Open Source DMX512 / RDM stack for PIC32 microcontrollers. It
+contains code to function as either an DMX / RDM controller (transmitter) or as an
+DMX receiver / RDM responder.
+
+# Features {#main-features}
+
+## USB to DMX / RDM Controller {#main-features-controller}
+
+- Configurable transmit timing parameters, including:
+ - Break duration
+ - Mark duration
+ - RDM response Timeout / Broadcast response timeout
+- Response timing information, including:
+ - Break time
+ - Mark time
+ - Response delay
+
+## DMX512 / RDM Receiver {#main-features-responder}
+
+- The ability to simulate many different types of @ref rdm-models.
+ Together, they support all PIDs from the E1.20, E1.37-1 & E1.37-2
+ standards. This allows RDM controllers to be tested without having access
+ to a large number of RDM devices. The code can act as:
+ - An LED Driver
```

Comment:
Can we make the capitalisation consistent, either drop driver, moving light, network device, or promote the others.